### PR TITLE
AL_Systems

### PR DESCRIPTION
**Pull request by aliang8**
Added test cases for systems of equations and sequences. 

Note: underscores (_) and brackets([) are not recognized by regex. to be used for subscripts (while writing sequences) and also lists

### CODE REVIEW

File: lib/parse.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ class Parser {
         }
 
         if (items.length > 1) {
-            if (items.every((item) => item.type === 'Relation')) {
+            if (items.every((item) => item.type === 'Apply')) {
```
Comment (k4b7) [inline lib/parse.js]: This should probably be a bit more restrictive.  It should also check that the `op` for each of these is one of `eq, lt, gt, le, ge`.

File: lib/parse.js [hunk 2]
```diff
@@ -471,5 +471,5 @@ const postProcess = (ast) => {
 const parser = new Parser();
 
 export default function parse(math) {
-  return postProcess(parser.parse(math));
```
Comment (k4b7) [inline lib/parse.js:474]: Delete `postProcess` as it isn't being used anymore.

File: test/parser_test.js [hunk 3]
```diff
@@ -130,6 +127,20 @@ describe("Parser.parse", () => {
...
+        'x + 1 = 2, x^2 + 2x + 1 = 0',
+        'a = b, b = c, c = d' 
+    ])
+
+    suite("sequences", [
+        '1, 1, 2, 3, 5',
+        'x, x + 1, x + 3',
+        'a, a^3, a^5',
+        //'r_1, r_2, r_3',
+    ])
```
Comment (k4b7) [inline test/parser_test.js:142]: Thanks for adding these tests.

**Review by k4b7 — CHANGES_REQUESTED** [review]

File: lib/parse.js (hunk 1 above)
Comment (aliang8) [reply lib/parse.js]: Yup, fixed. The `isRelation` function does just that.

**Review by k4b7 — APPROVED** [review]

**Pull request merged**